app/src/main/java/eu/faircode/email/WorkerCleanup.java

```diff
@@ -70,14 +70,14 @@ static void cleanup(Context context, boolean manual) {
                 // Check message files
                 Log.i("Checking message files");
                 List<Long> mids = db.message().getMessageWithContent();
+                File dir = new File(context.getFilesDir(), "messages");
+                if (!dir.exists())
+                    dir.mkdir();
                 for (Long mid : mids) {
-                    EntityMessage message = db.message().getMessage(mid);
-                    if (message != null) {
```

Comment:
This needs to stay because the message might be deleted in the meantime.

Reply:
Ah! I already had doubts if this was equivalent, thanks.